vuepress/docs/v6.3.2/docs/curate/ecr-bundle-details.md

```diff
@@ -486,6 +486,10 @@ This descriptor contains a list of Groups:
     - code: my_group # Identifies the Group
       name: "My group" # The name of the Group
 
+::: tip
+Groups in a content descriptor are configured by `mainGroup` and `groups` properties.
```

Comment:
Let's move this into the descriptor samples themselves. Examples here https://github.com/entando-ps/sme-celio-bundle/blob/master/contents/avv252-descriptor.yaml and https://github.com/entando-ps/sme-celio-bundle/blob/master/pages/sme_celio_avvisi-descriptor.yaml

For Content:
  mainGroup:  myGroup
  groups:                        
     - free

For Page (I think this is already there)
    ownerGroup: myGroup
    joinGroups: ['free']

Reply:
My formatting was lost but check the examples for that. The one remaining but would be to explain that the ownerGroup is for the group of users who can manage the entity in the AppBuilder where the join groups are those who can see or access the entity. The free group means it's a public entity. I'm not sure if that is explained somewhere already.

Reply:
@nshaw getting a 404 at the links ??

Reply:
Permissions since it's private repo. You and @jyunmitch should have an invite/access now.